org.jacoco.doc/docroot/doc/build.html

```diff
@@ -80,67 +79,34 @@ <h2>Testing with different JDKs</h2>
   &lt;toolchain&gt;
     &lt;type&gt;jdk&lt;/type&gt;
     &lt;provides&gt;
-      &lt;id&gt;java15&lt;/id&gt;
-      &lt;version&gt;1.5&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
+      &lt;id&gt;java5&lt;/id&gt;
+      &lt;version&gt;5&lt;/version&gt;
     &lt;/provides&gt;
     &lt;configuration&gt;
       &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.5&lt;/jdkHome&gt;
     &lt;/configuration&gt;
   &lt;/toolchain&gt;
+
   &lt;toolchain&gt;
     &lt;type&gt;jdk&lt;/type&gt;
     &lt;provides&gt;
-      &lt;id&gt;java16&lt;/id&gt;
-      &lt;version&gt;1.6&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
-    &lt;/provides&gt;
-    &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.6&lt;/jdkHome&gt;
-    &lt;/configuration&gt;
-  &lt;/toolchain&gt;
-  &lt;toolchain&gt;
-    &lt;type&gt;jdk&lt;/type&gt;
-    &lt;provides&gt;
-      &lt;id&gt;java17&lt;/id&gt;
-      &lt;version&gt;1.7&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
-    &lt;/provides&gt;
-    &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.7&lt;/jdkHome&gt;
-    &lt;/configuration&gt;
-  &lt;/toolchain&gt;
-  &lt;toolchain&gt;
-    &lt;type&gt;jdk&lt;/type&gt;
-    &lt;provides&gt;
-      &lt;id&gt;java18&lt;/id&gt;
-      &lt;version&gt;1.8&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
-    &lt;/provides&gt;
-    &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.8&lt;/jdkHome&gt;
-    &lt;/configuration&gt;
-  &lt;/toolchain&gt;
-  &lt;toolchain&gt;
-    &lt;type&gt;jdk&lt;/type&gt;
-    &lt;provides&gt;
-      &lt;id&gt;java19&lt;/id&gt;
-      &lt;version&gt;1.9&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
+      &lt;id&gt;java6&lt;/id&gt;
+      &lt;version&gt;6&lt;/version&gt;
     &lt;/provides&gt;
     &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.9&lt;/jdkHome&gt;
+      &lt;jdkHome&gt;/usr/lib/jvm/java-6-openjdk-amd64&lt;/jdkHome&gt;
     &lt;/configuration&gt;
   &lt;/toolchain&gt;
```

Comment:
I was a bit confused when reading this. Maybe a comment at the end would make clear that the file is not complete om respect to the description that is following:

    <!-- Add more JDK versions here -->

Reply:
I wanted to reduce boilerplate/duplication that we had here before. Comment sounds good to me.

Reply:
Added
```
<!-- you can specify other JDKs in this file -->
```